wagn/rails/controllers/card_controller.rb

```diff
@@ -73,6 +73,11 @@ def per_request_setup
     Card::Cache.renew
     Card::Env.reset :controller=>self
     Card::Auth.set_current_from_session
+    
+    if params[:id] && !params[:id].valid_encoding?  # slightly better way to handle encoding issues (than the rescue in load_id)
+                                                    # we should find the place where we produce these bad urls
```

Comment:
the problem are characters like é as in "L'oréal". they are part of card names but I can't find the place where the bad links are build. And it has nothing to do with recent changes. These errors occasionally occurred for months.